src/OctoshiftCLI.Tests/GithubClientTests.cs

```diff
@@ -619,6 +629,138 @@ await FluentActions
                 .WithMessage($"GitHub API error: {EXPECTED_RESPONSE_CONTENT}");
         }
 
+        [Fact]
+        public async Task PostGraphQLAsync_Returns_JObject_Response()
+        {
+            // Arrange
+            using var httpClient = new HttpClient(MockHttpHandlerForGraphQLPost().Object);
+            var githubClient = new GithubClient(_mockOctoLogger.Object, httpClient, null, _retryPolicy, _dateTimeProvider.Object, PERSONAL_ACCESS_TOKEN);
+
+            // Act
+            var response = await githubClient.PostGraphQLAsync("http://example.com", _rawRequestBody);
+
+            // Assert
+            Assert.True(JToken.DeepEquals(response, JObject.Parse(EXPECTED_GRAPHQL_JSON_RESPONSE_BODY)));
```

Comment:
We use fluent assertions, so either use `Should().BeTrue()` or IMO even better the following:
```suggestion
            response.ToJson().Should().Be(EXPECTED_JSON_REQUEST_BODY);
```

Reply:
Thanks for flagigng! Let me try this. I was trying to use an assertion like that but it was failing :(

Reply:
That works perfectly! Let me add it.